src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -6,22 +6,31 @@
     <AssemblyName>System.Threading.Tasks</AssemblyName>
     <AssemblyVersion>4.0.11.0</AssemblyVersion>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <!-- Monitor type-forward seen as duplicate type -->
```

Comment:
I'll check if I can disable it with a pragma. I just copied this over from the internal project and can't remember why I initially added it.